tests/request-builder.js

```diff
@@ -0,0 +1,132 @@
+import fetch from 'cross-fetch';
```

Comment:
```suggestion
```

Pode aproveitar para remover essa dependência. Só precisa adequar o teste `With CORS and Security Headers enabled`:

```js
      const responseHeaders = {};

      response.headers.forEach((value, key) => {
        responseHeaders[key] = [value];
      });
```

e aqui nesse arquivo

```js
    if (response.headers['Content-Type'] === 'application/x-ndjson') {
```

Reply:
Precisei mudar também a forma da leitura do stream. Não sei se daria para ser mais simples.